crates/bevy_ecs/src/world/unsafe_world_cell.rs

```diff
@@ -143,7 +143,7 @@ impl<'w> UnsafeWorldCell<'w> {
     /// below), which includes all entities, components, and resources. Notably, calling this on
     /// [`WorldQuery::init_fetch`](crate::query::WorldQuery::init_fetch) and
     /// [`SystemParam::get_param`](crate::system::SystemParam::get_param) are most likely *unsound* unless
-    /// you can prove that the underlying [`World`] is exclusive, which in normal circumstances is not.
+    /// you can prove that the underlying [`World`] is exclusive, which in normal circumstances it is not.
```

Comment:
I am not sure about this change. It feels like an awkward sentence but I'm not sure if the added "it" is helping.

Reply:
I’ve reverted this one